prep-html-skeleton/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+
+<html>
+  <title>HTML Skeleton Exercise</title>
+
+  <body>
+    <h1>I built a proper HTML skeleton!</h1>
+  </body>
+</html>
```

Comment:
It looks like you are missing a `<head>` tag. Your `<title>` should go inside of a `<head>`. 

It should look something like this

```
<!DOCTYPE html>
<html>
<head>
  <title>HTML Skeleton Exercise</title>
</head>
<body>
  <h1>I built a proper HTML skeleton!</h1>
</body>
</html>
```

Reply:
![prep html skeleton revision](https://github.com/PhilChoe/javascript-prep-0624-code-solutions/assets/172565484/b5459c1d-7f65-4d25-90a1-71ca3104de79)